src/Common/Doctrine/Type/LocaleType.php

```diff
@@ -18,13 +18,17 @@ public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $pla
     }
 
     /**
-     * @param string $locale
+     * @param string|null $locale
```

Comment:
Can't you type hint this inside the method signature?

Reply:
Not yet, only once php 7.3 is the minimum version of fork since the declaration wouldn't be the same as the base class.